app/livechat/server/lib/Livechat.js

```diff
@@ -603,6 +603,17 @@ export const Livechat = {
 		const visitor = LivechatVisitors.findOneById(room.v._id);
 		const agent = Users.findOneById(room.servedBy && room.servedBy._id);
 
+		const externalCustomFields = () => {
+			try {
+				const customFields = JSON.parse(settings.get('Accounts_CustomFields'));
+				return Object.keys(customFields)
+					.filter((customFieldKey) => customFields[customFieldKey].allowExternalSharing === true)
```

Comment:
we need to document the property `allowExternalSharing` right? so people can know it exists and use it too..

also, what about changing the name to make it more clear what it will cause, I thought something like `sendToIntegrations`?